examples/GEOME/test/ark-21547-Car2PIRE_0334-test.json

```diff
@@ -7,7 +7,7 @@
     "hasContextCategory": ["Marine Biome"],
     "hasMaterialCategory": ["Organic Material"],
     "hasSpecimenCategory": ["Whole Organism"],
-    "informalClassification": ["Gastropoda"],
+    "informalClassification": "Gastropoda",
```

Comment:
This should be a string not an array, per https://github.com/isamplesorg/metadata/blob/main/examples/GEOME/GEOMEiSamplesMapping.json

Reply:
It is a multi-valued property in iSampelsSchema0.5 though

Reply:
Hmmm, ok, I can convert it back.  I guess that's the source of the mismatch.